examples/recording_service/README.md

```diff
@@ -0,0 +1,23 @@
+# Recording Service
+
+RTI Recording Service includes three tools:
```

Comment:
Replace first paragraph with:

RTI Recording Service is the first off-the-shelf solution for reliable and non-intrusive recording, analysis and replay of real-time data, messages and events at high data rates and high fidelity. RTI Recording Service is comprised of three different tools: